applications/chats/serializers.py

```diff
@@ -16,13 +16,24 @@ class Meta:
 class RoomSerializer(serializers.ModelSerializer):
     class Meta:
         model = Room
-        fields = ('id', 'created_at', 'topic', 'receiver', )
-        read_only_fields = ('id', 'created_at', 'topic', 'receiver', )
+        exclude = ('members', )
+        read_only_fields = ('id', 'created_at', 'topic', 'receiver', 'unread_message_count', )
```

Comment:
No need for the last comma here. In exclude is necessary to indicate that is a tuple.
```suggestion
        read_only_fields = ('id', 'created_at', 'topic', 'receiver', 'unread_message_count')
```